### PR TITLE
[UIK-2631][docs] Replaced Validation example with link to Form, linted texts

### PR DESCRIPTION
**Pull request by sheila-semrush**
## Motivation and Context

Validation pattern example had the same functionality as Form example.

I removed Validation example from the Example page and replaced it with a link to Form.

Also fixed some small issues in text.

## Screenshots (if appropriate):

image

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue).
- [ ] New feature (non-breaking change which adds functionality).
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected).
- [x] Nice improve.

## Checklist:

- [x] My code follows the code style of this project.
- [x] I have updated the documentation accordingly or it's not required.
- [ ] Unit tests are not broken.
- [ ] I have added changelog note to corresponding `CHANGELOG.md` file with planned publish date.
- [ ] I have added new unit tests on added of fixed functionality.

### CODE REVIEW

**Comment by sheila-semrush** [comment]
@j-mnizhek I just replaced the whole example with a link to Form. What do you think?

Form examples improvements are still in progress/planned.

@Valeria-Zimnitskaya should I remove the example file whatsoever or do you need it?

**Review by j-mnizhek — APPROVED** [review]
Left one small comment. Everything other is good 😎

File: website/docs/patterns/validation-form/validation-form-code.md [hunk 1]
```diff
@@ -3,12 +3,6 @@ title: Validation
...
-
-::: sandbox
-
-<script lang="tsx">
-  export Demo from './examples/form-validation-example-by-`unfocus`.tsx';
-</script>
-
-:::
+::: info
+Refer to [Form](../form/form-code.md) for examples of form validation.
```
[the remark this answers is not in the document]
Comment (sheila-semrush) [reply website/docs/patterns/validation-form/validation-form-code.md:7]: (I also thought we might turn the plaque into plain text a bit later, when it's up long enough and had a chance to be seen by someone)

**Comment by Valeria-Zimnitskaya** [comment]
> @Valeria-Zimnitskaya should I remove the example file whatsoever or do you need it?
 lets keep it, i'll move it to the storybook

**Review by Valeria-Zimnitskaya — APPROVED** [review]

**Pull request merged**